_docs/getting-started.md

```diff
@@ -0,0 +1,28 @@
+# Getting Started
```

Comment:
It'd be good to add instructions on how to start the app, just to prove the install worked.

Reply:
Added a quick rundown of the most useful package scripts and how to run them - which includes 'how to start the app'.